### PR TITLE
Test Helm release

### PR DESCRIPTION
**Pull request by pereztr5**

### CODE REVIEW

**Comment by JAORMX** [comment]
Looks promising!

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - name: Checkout tag
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Publish Helm Chart
+        uses: tylerauerbeck/helm-gh-pages@main
+        with:
+          token: ${{ secrets.PAT }}
```
Comment (JAORMX) [inline .github/workflows/release.yaml:20]: do we need to use a PAT here? Can't we use the `$GITHUB_TOKEN` that's already available to the workflow?

Comment (pereztr5) [reply .github/workflows/release.yaml:20]: Oh, do you know which user that token is associated with? Also is it just `secrets.GITHUB_TOKEN`?

Comment (JAORMX) [reply .github/workflows/release.yaml:20]: @pereztr5 it's a temporary user that has the current workflow as the subject. You're able to add/remove permissions to it depending on what you wanna do [1]. I guess it really depends on where the target repo is.

[1] https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings

**Review by JAORMX — DISMISSED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (pereztr5) [reply .github/workflows/release.yaml]: @sfunkhouser got this token, `PUBLIC_REPO_GH_PAT`,  working for this repo.

**Comment by pereztr5** [comment]
@JAORMX Okay this is ready and tested.

**Review by JAORMX — APPROVED** [review]

**Pull request merged**